pkg/syncer/syncer.go

```diff
@@ -176,7 +176,7 @@ func getAllGVRs(discoveryClient discovery.DiscoveryInterface, resourcesToSync ..
 			return nil, err
 		}
 	}
-	gvrstrs := []string{"namespaces.v1."} // A syncer should always watch namespaces
+	gvrstrs := []string{"namespaces.v1.", "configmaps.v1."} // A syncer should always watch namespaces, serviceaccounts and configmaps.
```

Comment:
Maybe enabling syncing of the configmap (and the associated testing) can be accomplished in a separate PR?

Reply:
Done, will open a new pr for those changes with test.